ta/avb/entry.c

```diff
@@ -19,28 +22,42 @@ static TEE_Result get_slot_offset(size_t slot, size_t *offset)
 	return TEE_SUCCESS;
 }
 
-static TEE_Result create_state(uint32_t lock_state, TEE_ObjectHandle *h)
```

Comment:
I'd prefer keeping the old `create_state()` and `open_state()` functions as they are. If you think the old names are a bit confusing you can rename them to `create_rb_state()` and `open_rb_state()` respectively.

Reply:
renamed to `create_rb_state()`/`open_rb_state()`